modules/ROOT/pages/authentication-authorization/ldap-integration.adoc

```diff
@@ -97,9 +97,24 @@ runtime using xref:reference/procedures.adoc#procedure_dbms_setconfigvalue[`dbms
 
 First, you configure Neo4j to use LDAP as an authentication and authorization provider.
 
-. Uncomment the setting `dbms.security.auth_enabled=false` and change its value to `true` to turn on the security feature.
+. Make sure security is turned on.
+For this purpose uncomment the setting `dbms.security.auth_enabled=false` and change its value to `true`.
 . Uncomment the settings `dbms.security.authentication_providers` and `dbms.security.authorization_providers` and change their value to `ldap`.
 This way, the LDAP connector is used as a security provider for both authentication and authorization.
++
+Note these configuration values are comma-separated lists, so if you wish to continue to use native authentication and authorization alongside LDAP, then these providers can be added to the existing `native` provider.
+The providers will be queried in the order they are declared.
++
+.Configuration
+======
+[source,configuration,role="noheader"]
+----
+dbms.security.authentication_providers=ldap,native
+dbms.security.authorization_providers=ldap,native
+----
+This example contains a list of LDAP and Neo4j native providers to use for authentication.
```

Comment:
I don't think we need an explanation after the example. We either do it before or better as a caption.
```suggestion
```